src/Sylius/Bundle/ApiBundle/test/config/api_platform/config.yaml

```diff
@@ -0,0 +1,4 @@
+'%sylius.model.zone.class%':
+    collectionOperations:
+        admin_get (unset): ~
+        admin_post (unset): ~
```

Comment:
:-1: operations should not be merged but replaced and it'd solve everyone's issues.

Reply:
Do you mean, that we should replace the whole configuration of the resource? That would make sense. But the issue is, that for many cases Order would be replaced with the new configuration and once the configuration is replaced you will never receive upstream changes or features - therefore BC may be challenging on our side and it may raise some issues on the user side to determine what has changed in core and apply changes to his/hers app

Reply:
Yes really interesting, I hope that the next version of API Platform will help you to do so in an less constricting way! Btw are you doing Archtectural Decision Records for these kind of things?

Reply:
For sure we should add for this :+1: